### PR TITLE
Store prediction data for review

### PR DESCRIPTION
**Pull request by holyketzer**
Related to https://github.com/augerai/a2ml/issues/97

### CODE REVIEW

File: a2ml/tasks_queue/utils.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+def get_uid():
+    return uuid.uuid4().hex[:15].upper()
+
+def get_uid4():
+    return str(uuid.uuid4())
+
+def remove_dups_from_list(ar):
+    return list(OrderedDict.fromkeys(ar))
+
+def convertStringsToUTF8(params):
```
Comment (knobotz) [inline a2ml/tasks_queue/utils.py:15]: Let's remove this function and references to it, since it's a no-op.

File: a2ml/tasks_queue/utils.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                file_name+remote_file_info.get('file_ext'))
+        else:
+            local_file_path = os.path.join(local_dir,
+                remote_file_info.get('file_name') + remote_file_info.get('file_ext'))
+
+    if fsclient.isFileExists(local_file_path):
+        etag_changed = False
+        file_size_changed = False
+
+        #TODO: check etag to download new file, data_path_reloaded flag should be passed to auger_ml
```
Comment (knobotz) [inline a2ml/tasks_queue/utils.py:72]: References to auger-ml should be removed, along with this commented code.

File: a2ml/tasks_queue/data_source_api_pandas.py [hunk 3]
```diff
@@ -0,0 +1,1083 @@
+import arff
```
Comment (knobotz) [inline a2ml/tasks_queue/data_source_api_pandas.py:1]: Is this entire file necessary to port over? Could we just move over the parts that are necessary? Also, it's not covered by tests in this project.

**Comment by knobotz** [comment]
@holyketzer there are a couple of failing tests after merging w/ master. Please take a look.

**Comment by holyketzer** [comment]
> @holyketzer there are a couple of failing tests after merging w/ master. Please take a look.

fixed

**Pull request merged**